### PR TITLE
Add IDE messages for creating/deleting audio and video clips

### PR DESCRIPTION
**Pull request by peter-b**

### CODE REVIEW

File: docs/dictionary/message/deleteAudioclip.lcdoc [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+The actual deletion is not triggered by the <deleteAudioclip> <message>,
+so <trap|trapping> the <message> and not allowing it to <pass> does
+not prevent the <audioclip(object)> from being removed.
+
+However, the undo <command> will restore a <audioclip(object)> after
+it is deleted by the user.
+
+References: audioclip (object), delete (command), stack (object), trap
+	(glossary), button (glossary), message (glossary),
```
Comment (livecodeali) [inline docs/dictionary/message/deleteAudioclip.lcdoc]: at the moment references can't be wrapped between the name and type

**Comment by peter-b** [comment]
@livecodeali I've corrected the line wrapping in the documentation!

**Comment by livecodeali** [comment]
@livecode-vulcan review ok 461c919

**Comment by livecode-vulcan** [comment]
💙 review by @livecodeali ok 461c919605b4d0a04c4bab2942f2fbb8fc95f86e

**Comment by livecode-vulcan** [comment]
😎 test success 461c919605b4d0a04c4bab2942f2fbb8fc95f86e
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/2632)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/2642)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/2627)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/2643)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/2649)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/2655)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/2667)

**Pull request merged**